docs/static/arcsight-module.asciidoc

```diff
@@ -0,0 +1,406 @@
+
+* *DNS Data*
+** Dashboards: Microsoft DNS Overview
+** Data Types: Microsoft DNS devices
+
+[[network-dashboards-arsight]]
+===== Example Network Dashboards
+
+[role="screenshot"]
+image::static/images/arcsight-network-overview.png[Network overview dashboard]
+
+[role="screenshot"]
+image::static/images/arcsight-network-suspicious.png[Network suspicious activity dashboard]
+
+These Kibana visualizations enable you to quickly understand the top devices,
+endpoints, attackers, and targets. This insight, along with the ability to
+instantly drill down on a particular host, port, device, or time range, offers a
+holistic view across the entire environment to identify specific segments that
+may require immediate attention or action. You can easily discover answers to
+questions like:
+
+* Who are my attackers and what are they targeting?
+* Which of my devices or endpoints are the busiest and what services were
+rendered?
+* How many unique attackers, techniques, signatures, or targets were triggered
+at any given point in time?
+* What are the top sources, destinations, protocols, and behaviors that are
+causing the elevated count of failures?
+
+[[configuring-arcsight]]
+==== Configuring the Module
+
+You can specify additional options for the Logstash ArcSight module in the
+`logstash.yml` configuration file or with overrides through the command line
+like in the getting started. For more information about configuring modules, see
+<<logstash-modules>>.
+
+As an example, the following settings can be appended to `logstash.yml` to
+configure your module:
+
+[source,yaml]
+-----
+modules:
+  - name: arcsight
+    var.input.eventbroker.bootstrap_servers: “eb_host:39092”
+    var.input.eventbroker.topics: “eb_topic”
+    var.output.elasticsearch.hosts: "localhost:9200"
+    var.elasticsearch.username: "elastic"
+    var.elasticsearch.password: "changeme"
+    var.kibana.host: “localhost:5601”
+    var.kibana.username: "elastic"
+    var.kibana.password: "changeme"
+-----
+
+[[arcsight-module-config]]
+===== Logstash ArcSight Module Configuration Options
+
+These are the configurable settings available for the Logstash ArcSight module,
+along with the component of the module the setting will affect. When overriding
+settings in the command line, the setting option must be prefixed with the
+module name, i.e. `arcsight.var.inputs` instead of `var.inputs`.
+
+All settings are optional. If you don't specify configuration settings, Logstash
+uses the defaults.
+
+//ALVIN: I don't see a good reason to have the summary table plus the reference info, so I have not included the table. It doesn't add enough value to warrant the real estate it takes up. Plus...tables don't work well when we go from docbook > PDF. If you feel strongly about having it, you can submit a separate PR, but make sure you make each setting an active link, or users will be extra annoyed by the repeated info.
+
+*`var.inputs`*::
++
+--
```

Comment:
The formatting might look a bit unusual. The dashes are needed to include bulleted lists within definition lists.

(DLs are our standard way to format config options...not used consistently across the docs, but the recommended way.)